### PR TITLE
[DUOS-2950] GET DAA File API

### PR DESCRIPTION
**Pull request by aarohinadkarni**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2950

### Summary
Adds a new endpoint to get the actual file associated with a Data Access Agreement along with API documentation.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by rushtong** [comment]
Let's make sure this is rebased on the latest dev, in particular, we need https://github.com/DataBiosphere/consent/pull/2275 to fully test

File: src/main/java/org/broadinstitute/consent/http/service/DaaService.java [hunk 1]
```diff
@@ -101,4 +101,15 @@ public DataAccessAgreement findById(Integer daaId) {
     }
     throw new NotFoundException("Could not find DAA with the provided ID: " + daaId);
   }
+
+  public InputStream findFileById(Integer daaId) {
```
Comment (rushtong) [inline src/main/java/org/broadinstitute/consent/http/service/DaaService.java:105]: Can we add a service test for this method?

**Review by rushtong — APPROVED** [review]
Thank you for the changes! 👍🏽

**Review by fboulnois — APPROVED** [review]
tests and code looks good 👍

**Pull request merged**